src/Http/Requests/Newsletter/NewsletterUpdateRequest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests\Newsletter;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class NewsletterUpdateRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "email" => ["required"],
+            "type" => ["required"],
```

Comment:
Maybe we should validate the type more precisely? What I see, we have in Enum two types.